script/generate-signaling-protocol

```diff
@@ -26,4 +26,5 @@ end
 
   # workaround for the Long type import issue. Remove when https://github.com/protobufjs/protobuf.js/pull/1166/files is released.
   File.write("src/#{dir}/#{protocol}.d.ts", "import * as Long from \"long\";\n" + File.read("src/#{dir}/#{protocol}.d.ts"))
+  File.write("src/#{dir}/#{protocol}.js", "$util.Long = undefined;\n$protobuf.configure();", mode:"a")
```

Comment:
Should we add this to the top of the file rather than very bottom, similar to line 28?

Reply:
We cannot add it to the top of the file, We can either add it to the bottom or we have to add it to a specific line once protobuf.util is intialialized. We can try adding it at a specific line if it makes a difference once we are changing the proto types from Long to number.